pkg/cmd/dockerregistry/dockerregistry.go

```diff
@@ -137,7 +138,15 @@ func Execute(configFile io.Reader) {
 	app.Config.HTTP.Headers.Set("X-Registry-Supports-Signatures", "1")
 
 	app.RegisterHealthChecks()
-	handler := alive("/", app)
+	handler := http.Handler(app)
```

Comment:
I'd be ok with single quota for a set of methods / paths that only includes uploads, as long as we can identify up front (before we merge) that it addresses the issue at hand.  Future changes can well expand it.